controllers/cluster_controller_test.go

```diff
@@ -277,6 +278,127 @@ func TestClusterReconcilerDeleteNoCAPIClusterSuccess(t *testing.T) {
 	}
 }
 
+func TestSnowClusterReconcileSuccess(t *testing.T) {
```

Comment:
could we test this from `pkg/providers/snow/reconciler/reconciler_test.go`?

Reply:
Good idea, I'll try and see if it can fix the code coverage report